### ISSUE TITLE
API Key requested 30 days ago

### ISSUE THREAD

**Issue by mabalaru** [blockchain/api-v1-client-python#147]
Hello Blockchain.com dev team. 

Support sent me here because I am waiting for more then 30 days to receive an API key.

I have requested from this link https://api.blockchain.info/customer/signup. Standard message pops up that it will take 2-3 days but nothing happens. Sent message to blockchain support team and they pointed fingers to this place.

My app needs this so that it can integrate the bitcoin price from your API. 

Please be so kind and help me with this. 

Thank you in advance

**Comment by divinationstar** [comment]
Has anyone responded to you regarding this? I'm having issues with their API not sending callbacks to our URL string for payment confirmation (at 4 confirmations). It worked for years and then stopped working last 1-2 weeks.

**Comment by mabalaru** [comment]
No, no response from their side .. and it's opened 30 days ago.
